dist/css/style.css

```diff
@@ -217,43 +242,95 @@ li {
   line-height: 28px;
 }
 
+@media (min-width: 768px) {
+  .header-logo {
+    display: none;
+  }
+}
+
+.nav__list {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  ul.nav__list {
+    display: flex;
+    gap: 8px;
+    list-style: none;
+    margin: 0 auto;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav__item {
+    padding: 10px 12px;
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+}
+
 .header-hamburger {
   padding: 8px;
 }
 
+@media (min-width: 768px) {
+  .header-hamburger {
+    display: none;
+  }
+}
+
 /* - - - - - - - - - - - - - - - - - - - - - -
 - - - - - - - - Headline section - - - - - - - -
 - - - - - - - - - - - - - - - - - - - - - - */
 #headline {
   padding: 64px 0;
-  background-image: url("../images/headline/background-1.svg"),
-    url("../images/headline/background-2.svg"),
-    url("../images/headline/background-3.svg"),
-    url("../images/headline/background-4.svg"),
-    url("../images/headline/background-5.svg"),
-    url("../images/headline/background-6.svg"),
-    url("../images/headline/background-7.svg"),
-    url("../images/headline/background-8.svg");
-  background-position: center center, 100% 42%, 0% 72%, 0% 19%, 0% 0%, 100% -1%,
-    40% 68%, 100% 90%;
+  background-image: url("../images/headline/background-1.svg"), url("../images/headline/background-2.svg"), url("../images/headline/background-3.svg"), url("../images/headline/background-4.svg"), url("../images/headline/background-5.svg"), url("../images/headline/background-6.svg"), url("../images/headline/background-7.svg"), url("../images/headline/background-8.svg");
+  background-position: center center, 100% 42%, 0% 72%, 0% 19%, 0% 0%, 100% -1%, 40% 68%, 100% 90%;
   background-repeat: no-repeat;
   background-size: cover, auto, auto, auto, auto, auto, auto, auto;
 }
 
+@media (min-width: 768px) {
+  #headline {
+    background-image: none;
+  }
+  .headline__bg {
+    background-image: url("../images/Header-llustration-desktop@2x.png");
+    position: absolute;
+    z-index: 1000;
+  }
+}
+
 /* background-position: center center, 100% 48%, 2% 27%, 0 91%, 95% 96%, 57% 0; */
 .headline-inner {
```

Comment:
Dear Shahadat, I would like to suggest you avoid leaving blocks of code commented. According to the best practices.